Dockerfile

```diff
@@ -25,16 +25,10 @@ RUN pip uninstall easypqp \
     && pip install git+https://github.com/Nesvilab/easypqp.git@master \
     && pip install lxml
 	
-# set the working directory to the HOME of root
-WORKDIR /root
-
-# create directories
-RUN mkdir tmp
```

Comment:
I think when I was testing the Dockerfile, FragPipe crashed because there is no `tmp` directory. That was why I added this command. Can't remember the exact error message and the reason. If there is no harm, I suggest keeping it.